### PR TITLE
Hotjar ingtegration

### PR DESCRIPTION
**Pull request by lukeromanowicz**
### Related issues

https://github.com/DivanteLtd/vue-storefront/issues/840

### Short description and why it's useful

Added hotjar snippet as a module. Loads only on browser side. This feature also required to fix isServer flag in module registration

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)
- [x] I read the [TypeScript Action Plan](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/TypeScript%20Action%20Plan.md) and adjusted my PR according to it
- [x] I read about [Vue Storefront Modules](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/api-modules/about-modules.md) and I am aware that every new feature should be a module

### CODE REVIEW

File: src/modules/index.ts [hunk 1]
```diff
@@ -62,6 +63,7 @@ export const registerModules: VueStorefrontModule[] = [
   PromotedOffers,
   Magento2CMS,
   // GoogleAnalytics,
+  Hotjar,
```
Comment (filrak) [inline src/modules/index.ts]: Please keep it commented by default so it's not bundled while not needed

Comment (lukeromanowicz) [reply src/modules/index.ts]: @filrak done

**Comment by Nuneshere** [comment]
thanks @lukeromanowicz, you work is amazing! just on the hotjar website i try to verify the page, but even though the verification pop up shows up on my vue-storefront project saying that the hotjar is ok, the hotjar website still apoint to "Hotjar installation could not be verified, please confirm that the tracking code has been installed on this page.". Do you have any ideia on how to fix that?

**Comment by lukeromanowicz** [comment]
@Nuneshere Hi, thanks for testing this feature. Have you configured your hotjar id in the config file? Are you sure that your website is public and Hotjar can succesfully access it?

**Comment by Nuneshere** [comment]
@lukeromanowicz hey Luke, so I have been successful put the I'd on the config file as you oriented, my website is under Dev so it is on the localhost, but hotjar seems to be able to accept sites like that too ( they mentioned it on their site ). I checked in the inspect element on my localhost and the site included a hotjar link on the head of the site. I've no clue on why it's no working, on your tests have you successfully put it on work on the hotjar ? Do you run it locally or in public

**Pull request merged**